charts/vineyard/templates/worker/statefulset.yaml

```diff
@@ -0,0 +1,165 @@
+            {{- if eq (include "vineyard.checkTieredStore" .) "true" }}
+            --spill_path {{ include "vineyard.spill.path" . }}
+            --spill_lower_rate {{ include "vineyard.spill.lowerRate" . }}
+            --spill_upper_rate {{ include "vineyard.spill.upperRate" . }}
+            {{- end }}
+          env:
+          - name: VINEYARDD_NAME
+            value: {{ template "vineyard.fullname" . }}-vineyardd
+          - name: VINEYARDD_NAMESPACE
+            value: {{ .Release.Namespace }}
+          - name: MY_NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: MY_POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: MY_UID
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.uid
+          - name: MY_POD_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+          - name: MY_HOST_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+          - name: USER
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          {{- range $key, $value := .Values.worker.env }}
+          - name: "{{ $key }}"
+            value: "{{ $value }}"
+          {{- end }}
+          {{- if .Values.worker.resources  }}
+{{ include "vineyard.worker.resources" . | indent 10 }}
+          {{- end }}
+          ports:
+            - name: rpc
+              containerPort: {{ int .Values.worker.ports.rpc }}
+              protocol: TCP
+          livenessProbe:
+            tcpSocket:
+              port: {{ int .Values.worker.ports.rpc }}
+            periodSeconds: 60
+          readinessProbe:
+            exec:
+              command:
+              - ls
+              - /var/run/vineyard/vineyard.sock
+            initialDelaySeconds: 15
+            periodSeconds: 10
+            failureThreshold: 10
+          volumeMounts:
+          - name: vineyard-socket-volume
+            mountPath: /var/run/vineyard
+          {{- if eq (include "vineyard.checkTieredStore" .) "true" }}
+{{- include "vineyard.worker.tieredstoreVolumeMounts" . }}
+          {{- end }}
+      volumes:
+      {{- if eq (include "vineyard.checkTieredStore" .) "true" }}
+{{- include "vineyard.worker.tieredstoreVolume" . }}
+      {{- end }}
+      - name: vineyard-socket-volume
+        hostPath:
+          path: /runtime-mnt/vineyard/{{ .Release.Namespace }}/{{ $fullName }}
+          type: DirectoryOrCreate
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: app.kubernetes.io/instance
```

Comment:
The label `app.kubernetes.io/instance` seems missing in Vineyard worker's Pod metadata?

Reply:
That's my fault. Thanks for pointing out.